WildFly_28.0.1.Final/microprofile-6.0-selected-specifications/microprofile-6.0-selected-specifications-certification.adoc

```diff
@@ -0,0 +1,37 @@
+= TCK Results
+:ext-relative: {outfilesuffix}
+:imagesdir: ../../images/
+
+image:splash_wildflylogo_small.png[WildFly, align="center"]
+
+[[introduction]]
+== Introduction
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License],
+following is a summary of the TCK results for selected specifications from the release of
+https://github.com/eclipse/microprofile/releases/tag/6.0[MicroProfile 6.0]. This summary contains
+all specifications defined in https://github.com/eclipse/microprofile/releases/tag/6.0[MicroProfile
+6.0] except for MicroProfile Metrics which is not implemented by WildFly. WildFly does not claim to be a
+compatible implementation of https://github.com/eclipse/microprofile/releases/tag/6.0[MicroProfile
+6.0].
+
+=== WildFly, Certification Summary
+----
+Date:  Tue May 23 09:10:39 AM CEST 2023
```

Comment:
Is this the date at time of creating the certification summary or should it be the WildFly 28.0.1.Final release date (May 18, 2023)?

Reply:
The date of creating the certification summary, so it should be like this.